integration-tests/pom.xml

```diff
@@ -444,6 +444,12 @@
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Sorry for the confusion, @AmatyaAvadhanula ! I didn't realize that these tests were using TestNG.

You can leave out the `Assume` condition that I had suggested as that is the only thing that requires junit. No point having dependencies on both junit and TestNG.